### PR TITLE
[MadNLPGPU] Bug fix for empty Hessian

### PR DESCRIPTION
**Pull request by sshin23**
This PR fixes the bug of throwing integer division error when dealing with empty Hessian

### CODE REVIEW

File: lib/MadNLPTests/src/MadNLPTests.jl [hunk 1]
```diff
@@ -331,6 +331,27 @@ function eigmina(optimizer_constructor::Function; Arr = Array)
     end
 end
 
+function lp(optimizer_constructor::Function; Arr = Array)
```
Comment (frapac) [inline lib/MadNLPTests/src/MadNLPTests.jl:334]: I would maybe add a comment referring to the github issue to explain why we have this test. How about renaming the function `lp_issueXXX` with `XXX` the id of the github issue?

Comment (frapac) [inline lib/MadNLPTests/src/MadNLPTests.jl:334]: I would maybe add a comment referring to the github issue to explain why we have this test. How about renaming the function `lp_issueXXX` with `XXX` the id of the github issue?

**Review by frapac — APPROVED** [review]
LGTM!

File: lib/MadNLPTests/src/MadNLPTests.jl (hunk 1 above)
Comment (sshin23) [reply lib/MadNLPTests/src/MadNLPTests.jl]: Changed the name to `lp_examodels_issue75`

**Pull request merged**